schema.sql

```diff
@@ -8,3 +8,16 @@ create table animals (
     neutered boolean,
     weight_kg decimal
 )
+
+alter table animals add species varchar(50);
+
+update animals set species = 'unspecified';
+
+update animals set species = 'digimon' where name like '%mom';
+
+update animals set species = 'pokemon' where species is null;
+
+delete from animals;
+delete from animals where date_of_birth > date '2022-01-01';
+update animals set weight_kg = weight_kg * -1;
+update animals set weight_kg = weight_kg * -1 where weight_kg < 0;
```

Comment:
Everything that is a query that helps to analyze data should be saved in the queries.sql file. Please move this part of the code to the queries.sql file